test/extended/oauth/client_secret.go

```diff
@@ -113,6 +116,31 @@ var _ = g.Describe("[sig-auth][Feature:OAuthServer]", func() {
 	})
 })
 
+func getOAuthEndpoint(apiserverURL string) (string, error) {
+	requestURL := strings.Join([]string{apiserverURL, ".well-known/oauth-authorization-server"}, "/")
+	request, err := http.NewRequest(http.MethodGet, requestURL, nil)
+	if err != nil {
+		return "", err
+	}
+	client := &http.Client{Transport: &http.Transport{TLSClientConfig: &tls.Config{InsecureSkipVerify: true}}}
```

Comment:
Needs Proxy as well

Reply:
I changed it to use the existing method and am assuming that will use a proxy if needed